gemfiles/Gemfile.rails52

```diff
@@ -10,3 +10,4 @@ group :remote_test do
 end
 
 gem 'actionpack', '~> 5.2.0'
+gem 'money', '< 7.0.0'
```

Comment:
All of these should now be `gem 'money', '~> 6.8'` since we make assumptions here about the public interface of any Money gem, and don't get a 3.x version by surprise.